requirements.txt

```diff
@@ -3,3 +3,4 @@ openai
 unstructured
 faiss-cpu
 gradio
+tiktoken
```

Comment:
This does not seem to be used anywhere

Reply:
When running the app there is an error for not having this package and after adding it, the app ran just fine